src/Nethermind/Nethermind.AuRa.Test/Contract/TxPriorityContractTests.cs

```diff
@@ -93,7 +94,7 @@ public async Task whitelist_should_return_correctly()
             whiteListInContract.Should().BeEquivalentTo(expected);
         }
         
-        [Test]
+        [Todo("Needs to be fixed, configuration error in master.")]
```

Comment:
We need to uncomment it. These tests are flaky, but we can't mark it with to do.